Resources/doc/version_listener.rst

```diff
@@ -0,0 +1,150 @@
+Version listener and API versioning
+===================================
+Why version your API ?
+----------------------
+If you want to introduce breaking changes in your API, you should think about version your API. Indeed, if you introduce them on your current API, your users won't necessary upgrade their application directly and their applications won't work.
+
+How to version your API ?
+-------------------------
+There are several ways, none is standard :
+    - Include the version in the uri ``/v1/users``
+    - Use a query parameter ``/users?version=v1``
+    - Use a custom mime-type with an ``Accept`` header ``Accept: application/json; version=1.0``
+    - Use a custom header ``X-Accept-Version: v1``
+
+The ``FOSRestBundle`` allows you to use several of them at the same time or to choose one of them.
+
+Configure ``FOSRestBundle`` to use the api versioning
+-----------------------------------------------------
+You should activate the versioning in your config.yml:
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+    fos_rest:
+        versioning: true
+
+If you do not want to allow all the methods described above, you should choose which version resolver to enable:
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+    fos_rest:
+        versioning:
+            enabled: true
+            resolvers:
```

Comment:
i think it would also be good to define the order of the resolvers similar to here:
https://github.com/lunetics/LocaleBundle/blob/master/Resources/doc/index.markdown#guessers